File: apier-auth/src/main/java/cn/apier/auth/config/AuthConfig.java
package cn.apier.auth.config;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

/**
 * Created by yanjunhua on 2017/5/25.
 */
@ConfigurationProperties(prefix = "apier.auth")
@Configuration
public class AuthConfig
{
    private long timeWindowInMs;


    public long getTimeWindowInMs()
    {
        return timeWindowInMs;
    }

    public void setTimeWindowInMs(long timeWindowInMs)
    {
        this.timeWindowInMs = timeWindowInMs;
    }
}

File: apier-auth/src/main/kotlin/cn/apier/auth/common/AuthConfig.kt
package cn.apier.auth.common

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties(prefix = "apier.auth")
class AuthConfig {
    var tokenExpiredTimeInMs: Long = 3600000
    var loginExpiredTimeInMs: Long = 3600000

}